00-Basics/12-IsPrime/isPrime.cpp

```diff
@@ -0,0 +1,34 @@
+#include<iostream>
+#include<conio.h>
+using namespace std;
+
+int main(){
+	int size;
+	int a=0;
+     int tab[size];
```

Comment:
the value for `size` is not defined here. So again, this should not work, or result in undefined behaviour. And again, you don't need the array.